### PR TITLE
DX-2563

### PR DESCRIPTION
**Pull request by DX-Bandwidth**
Update API specs from upstream api-specs repository. Opened By: @ajrice6713

### CODE REVIEW

File: site/specs/global.yml [hunk 1]
```diff
@@ -203,7 +246,7 @@ info:
 
     |`VERIFICATION_NOT_YET_REQUESTED`|`517`|No request for verification was sent
     for this regulation address. It must first be verified before it is possible
-    to link DIDs.|
+    to link dids.|
```
Comment (bpateldx) [inline site/specs/global.yml:249]: Keep DID in CAPS?

File: site/specs/global.yml [hunk 2]
```diff
@@ -514,8 +557,8 @@ info:
     |`PROPAGATION_EXCEPTION`|`315`|The server encountered an error while
     applying the configuration options. Some of the configuration might have
     been applied successfully. You can safely execute the configuration
-    operation again, if the problem persists please contact the Bandwidth
-    support team.|
+    operation again, if the problem persists please contact the voxbone support
```
Comment (bpateldx) [inline site/specs/global.yml:560]: Keep voxbone? If yes, capitalize it. Suggest same for five more references below-:)

File: site/specs/voice.yml [hunk 3]
```diff
@@ -0,0 +1,5173 @@
...
+servers:
+  - url: https://voice.bandwidth.com/api/v2
+    description: Production
+paths:
+  /accounts/{accountId}/calls:
+    post:
+      tags:
+        - Calls
+      summary: Create Call
+      description: >-
```
Comment (bpateldx) [inline site/specs/voice.yml:24]: Typo: '>' -> '|'

File: site/specs/voice.yml [hunk 4]
```diff
@@ -0,0 +1,5173 @@
...
+          $ref: '#/components/responses/unsupportedMediaTypeError'
+        '429':
+          $ref: '#/components/responses/tooManyRequestsError'
+        '500':
+          $ref: '#/components/responses/internalServerError'
+      x-codeSamples:
+        - lang: cURL
+          source: |
+            curl 'https://voice.bandwidth.com/api/v2/accounts/12345/calls' \
+                -X POST \
```
Comment (bpateldx) [inline site/specs/voice.yml:64]: There might be som e history here-:) Why are we providing sample code within the spec?

Comment (ajrice6713) [reply site/specs/voice.yml:64]: The sample code gets automatically injected into the spec file and rendered in our documentation 

you can see an example of that here: 
https://dev.bandwidth.com/apis/voice/#tag/Calls/operation/createCall

Screen Shot 2022-06-21 at 4 44 36 PM

the code samples are all contained in this repository: 
https://github.com/Bandwidth/api-docs-code-snippets

File: site/specs/voice.yml (hunk 3 above)
Comment (ajrice6713) [reply site/specs/voice.yml:24]: Huh - these are all `|-` in my api-specs PR

Suspecting this has to do with the logic that adds in the code to the spec - we convert YAML to JSON and back to YAML

Let me investigate this

**Comment by DX-Bandwidth** [comment]
Corresponding Pull Request on [api-specs](https://github.com/Bandwidth/api-specs/pull/173) was merged.

**Review by matthewkmartin — APPROVED** [review]

**Pull request merged**